frontend/src/components/Forms/AddPermitForm.js

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import PropTypes from "prop-types";
+import { Field, reduxForm, formValueSelector } from "redux-form";
+import RenderField from "@/components/common/RenderField";
+import RenderSelect from "@/components/common/RenderSelect";
+import RenderDate from "@/components/common/RenderDate";
+import { Form, Button, Col, Row, Popconfirm, Tabs } from "antd";
+import * as FORM from "@/constants/forms";
+import { required, dateNotInFuture } from "@/utils/Validate";
+import { resetForm } from "@/utils/helpers";
+import { getDropdownPermitStatusOptions } from "@/selectors/staticContentSelectors";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  permitStatusOptions: PropTypes.arrayOf(CustomPropTypes.dropdownListItem).isRequired,
+  title: PropTypes.string.isRequired,
+  submitting: PropTypes.bool.isRequired,
+  permitTypeCode: PropTypes.string,
+  permitActivityTypeCode: PropTypes.string,
+};
+
+const defaultProps = {
+  permitTypeCode: "",
+  permitActivityTypeCode: "",
+};
+
+const permitTypes = [
+  {
+    label: "Coal",
+    value: "C",
+  },
+  {
+    label: "Mineral",
+    value: "M",
+  },
+  {
+    label: "Placer",
+    value: "P",
+  },
+  {
+    label: "Sand & Gravel",
+    value: "G",
+  },
+  {
+    label: "Quarry",
+    value: "Q",
+  },
+];
+
+const permitActivityTypes = [
+  {
+    label: "Operation",
+    value: "",
+  },
+  {
+    label: "Exploration",
+    value: "X",
+  },
+];
+
+const selector = formValueSelector(FORM.ADD_PERMIT);
+const { TabPane } = Tabs;
+
+const validate = (values) => {
```

Comment:
Maybe rename this? This is a little ambiguous as we already use validate for our forma validation (see line 99)

Reply:
Renamed to validateBusinessRules.